integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -7692,8 +7692,12 @@ msgstr ""
 "aufgetreten."
 
 #: xliff/utils.py
-msgid "Page {} could not be imported successfully because of the errors: {}"
-msgstr "Die Seite {} konnte nicht importiert werden aufgrund der Fehler: {}"
+msgid "Page "
+msgstr "Seite"
+
+#: xliff/utils.py
+msgid " could not be imported successfully because of the errors: "
+msgstr " konnte nicht importiert werden aufgrund der Fehler: "
```

Comment:
```suggestion
msgid "Page {} could not be imported successfully because of the errors:"
msgstr "Seite {} konnte nicht importiert werden aufgrund der Fehler:"
```

Reply:
Sorry, I noticed this after CircleCI Check 😓 It should be now fixed.